### PR TITLE
Correct the CHANGELOG-1.9.md

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
/cc @brendandburns
thanks

**Comment by jianglingxia** [comment]
/assign @bgrant0607
thanks

File: CHANGELOG-1.9.md [hunk 1]
```diff
@@ -559,7 +559,7 @@ SIG Azure worked on improvements in the cloud provider, including significant wo
...
+SIG Cluster Lifecycle has been focusing on improving kubeadm in order to bring it to GA in a future release, as well as developing the [Cluster API](https://github.com/kubernetes/kube-deploy/tree/master/ext-apiserver). For kubeadm, most new features, such as support for CoreDNS, IPv6 and Dynamic Kubelet Configuration, have gone in as alpha features. We expect to graduate these features to beta and beyond in the next release. The initial Cluster API spec and GCE sample implementation were developed from scratch during this cycle, and we look forward to stabilizing them into something production-grade during 2018.
```
Comment (bgrant0607) [inline CHANGELOG-1.9.md]: What's the reason for this change? We don't update old release notes. In any case, that path doesn't currently exist.

Comment (jianglingxia) [reply CHANGELOG-1.9.md]: sorry，a few days ago I checked that the link error but I just checked corrected,then I reupdated it,for your approve!thanks

**Comment by jianglingxia** [comment]
PTAL @bgrant0607 
thanks
